controller/engine_image_controller.go

```diff
@@ -829,6 +829,20 @@ func (ic *EngineImageController) createEngineImageDaemonSetSpec(ei *longhorn.Eng
 								TimeoutSeconds:      datastore.PodProbeTimeoutSeconds,
 								PeriodSeconds:       datastore.PodProbePeriodSeconds,
 							},
+							LivenessProbe: &v1.Probe{
+								ProbeHandler: v1.ProbeHandler{
+									Exec: &v1.ExecAction{
+										Command: []string{
+											"sh", "-c",
+											"/data/longhorn version --client-only",
+										},
+									},
+								},
+								InitialDelaySeconds: datastore.PodProbeInitialDelay,
+								TimeoutSeconds:      datastore.PodProbeTimeoutSeconds,
+								PeriodSeconds:       datastore.PodProbePeriodSeconds,
+								FailureThreshold:    datastore.PodLivenessProbeFailureThreshold,
```

Comment:
I think we can make time-related settings of `ReadinessProbe` to align with these values as well.